homeassistant/components/amberelectric/coordinator.py

```diff
@@ -0,0 +1,112 @@
+"""Amber Electric Coordinator."""
+from __future__ import annotations
+
+from datetime import timedelta
+from typing import Any
+
+from amberelectric import ApiException
+from amberelectric.api import amber_api
+from amberelectric.model.actual_interval import ActualInterval
+from amberelectric.model.channel import ChannelType
+from amberelectric.model.current_interval import CurrentInterval
+from amberelectric.model.forecast_interval import ForecastInterval
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import LOGGER
+
+
+def is_current(interval: ActualInterval | CurrentInterval | ForecastInterval) -> bool:
+    """Return true if the supplied interval is a CurrentInterval."""
+    return interval.__class__ == CurrentInterval
+
+
+def is_forecast(interval: ActualInterval | CurrentInterval | ForecastInterval) -> bool:
+    """Return true if the supplied interval is a ForecastInterval."""
+    return interval.__class__ == ForecastInterval
```

Comment:
This is the Python way:
```suggestion
    return isinstance(interval, ForecastInterval)
```